File: adventofcode/2018/2/p1.py
from collections import Counter

with open('./input') as f:
    lines = f.readlines()
    twos = 0
    threes = 0
    for line in lines:
        c = Counter(line)
        values = c.values()
        if 2 in values:
            twos +=1
        if 3 in values:
            threes +=1

    print(twos)
    print(threes)
    print(twos*threes)

File: adventofcode/2018/2/p1.jl
file = open("input")
lines = readlines(file)

function solve()
  twos = 0
  threes = 0
  for line in lines
    d = Dict{Char,Int64}()
    for char in line
      d[char] = get(d, char, 0) + 1
    end
    v = values(d)
    if 2 in v
      twos +=1
    end
    if 3 in v
      threes +=1
    end
  end
  println(twos)
  println(threes)
  twos*threes
end
print(solve())
